src/frontend/hooks/server/projects.ts

```diff
@@ -28,19 +19,29 @@ export function useGetOwnRole() {
   return useOwnRoleInProject({projectId});
 }
 
-export function useGetRemoteArchives() {
```

Comment:
noticed that this hook was being used to figure out if there's an active remote archive member. in some cases, the usages were slightly incorrect because they didn't account for the role (`useManyMembers()` currently doesn't do _any_ filtering of members).

opted for a more focused and succinct hook (`useActiveArchiveServer()`) that better addresses the needs of the existing usages.